gemfiles/rails_4.1.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
```

Comment:
These are simple gemfiles: could they be maintained by hand and Appraisal dispensed with?

Reply:
Yes, but, Appraisal also also gives you the ability to (for example) `bundle exec appraisal rake test` to run the test suite against all gemfiles at once.  That's super helpful when you're trying to keep a library working against a range of dependency versions, as is the case here.

It's certainly a "nice to have" and I can back that out if that's what it takes to get this merged.  I'm not aware of any projects adding Appraisal then regretting it and backing it out, FWIW.

Reply:
https://github.com/activeadmin/activeadmin/commit/b3d3ccb4232eb9edc99740ddc4d875482d93dc4e FWIW. That was @deivid-rodriguez's call, here I think it is up to @rafaelfranca

Reply:
Yup, I don't think `appraisal` provides a lot of benefit.

If a task like `bundle exec appraisal rake test` was to be run frequently by developers, it'd be a matter of adding a simple `Rakefile` task that iterates over the available gemfiles and runs tests on each using the proper `BUNDLE_GEMFILE` environment.

Reply:
I don't want to call out other projects by name for making a mess, but I've worked on gems that started with a "simple rake task" and grew it into a collection of fragile, unmaintanable rake tasks as they reinvented all the Appraisal functionality.

Reply:
Each to their own, I guess :)